RELEASE_NOTES.rst

```diff
@@ -1,3 +1,133 @@
+
+This release was brought to you by 65 contributors contributing 141
+commits
+
+The name of this release is BrewBear to highlight the improvements made to
+out dependency management system, including the addition of ``brew`` support
+for mac users. Mac has been an officially supported platform for a long time
+and we would like to draw attention to this often overlooked fact.
+Support has also been added for ``Conda`` and ``cargo`` allowing bears which
+use linters available from those package managers to manage their dependencies.
+
+Initial next generation core code has been merged and will be integrated over
+the coming releases. Until then, please use the current API.
+
+We have added a release candidate system to our release process to allow users
+to easily test changes in the latest pre-release version of coala.
+Instructions for obtaining the rc package will be posted for each minor/major
+release to the coala gitter channel, blog, and twitter. We encourage all users
+interested in checking out new features or helping to test these pre-releases
+on their projects. If you do find a regression in a release candidate, please
+report it to the issue tracker with the word ``Regression`` somewhere in the
+title.
+
+**Breaking Changes**
+
+- coala exit codes have changed. If you relied on coala in scripts or
+  other automated systems please check if it it still works with the new
+  behavior. You can find the new exit codes
+  `here <http://docs.coala.io/en/latest/Users/Exit_Codes.html>`__.
+  `PR #3905 <https://github.com/coala/coala/pull/3905>`_
+
+**General**
+
+- Section Inheritance had been added to the coafile parsing. With this
+  change we are deprecating implicit ``default`` section inheritance.
+  Instead, you can now have definite section inheritance by using the
+  ``.`` in the section title. Append operators are now supported allowing
+  addition of values to inherited section. See the ``Mockup`` section of
+  `cEP-0005 <https://github.com/coala/cEPs/blob/master/cEP-0005.md>`_
+  for a syntax example and `PR #3178 <https://github.com/coala/coala/pull/3178/>`_
+  for more information.
+- The section name ``cli`` is reserved for cli arguments and should not be
+  used in the coala configuration file (``coafile``). If you are currently
+  using it in a configuration file, coala will warn you.
+- coala now outputs absolute files paths when a bear fails. This should
+  make debugging bear failures easier when using custom bears.
+
+**Usability**
+
+- A new placeholder, ``source_lines``,  has been added to the coala
+  ``--format`` option. Use it to output the affected lines in the source file
+  when a result occurs.
+- Bears are displayed alphabetically when listed using ``--show-bears``.
+- coala outputs a warning if a setting is overridden in the same section
+  since this is probably not what the user intended.
+- ``--format`` takes precedence over ``--ci`` when both arguments are issued
```

Comment:
Will coala behave properly? I.e. not ask me for missing parameter via stdin and error out directly? If not, that's counterintuitive, if so the remote description is wrong because it's not format taking precedence but both being considered correctly.